src/main/kotlin/lotto/model/LottoPrize.kt

```diff
@@ -0,0 +1,12 @@
+package lotto.model
+
+enum class LottoPrize(val matchCount: Int, val prizeMoney: Int) {
+    THREE_MATCH(3, 5000),
+    FOUR_MATCH(4, 50000),
+    FIVE_MATCH(5, 150000),
+    SIX_MATCH(6, 2000000000);
```

Comment:
큰 수는 `_` 를 사용하여 가독성을 올려봅시다.
```suggestion
    SIX_MATCH(6, 2_000_000_000);
```